### PR TITLE
Toggle mobile menu feature-PULL REQUEST#8

### PR DESCRIPTION
**Pull request by efmomoh**
In this feature, I have added a toggling effect to the mobile hamburger menu, an ESLint file, a javascript file, and an Updated README.md file.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @efmomoh , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] Doesn't have linters set up properly. Since this project include javascript please make sure to also include `Eslint` in to your linter configurations. Use this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to set up appropriate linter files
- [ ] Doesn't have contribution from coding partner. This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode). Please make sure your coding partner contributes into your repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -547,16 +550,72 @@ label {
...
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
```
Comment (emmiiorji) [inline style.css]: Kindly ensure that the design of the menu bar is as given by the design.

Your design:
image

Figma Design:
image

File: index.html [hunk 2]
```diff
@@ -17,12 +17,17 @@
...
       <a class="efm" id="efm-2" href="#">EFM</a>
       <nav class="navbar">
-        <div class="bars"><a href="#"><i class="fa fa-bars"></i></a></div>
         <ul class="menu-list">
-          <li><a class="menu" href="">Portfolio</a></li>
-          <li><a class="menU" href="">About</a> </li>
-          <li><a class="menu" href="">Contact</a></li>
+          <li><a class="menu" href="#">Portfolio</a></li>
+          <li><a class="menu" href="#">About</a> </li>
+          <li><a class="menu" href="#">Contact</a></li>
```
Comment (emmiiorji) [inline index.html]: Kindly ensure that the correct sections of the page are navigated to when the menu items get clicked.

File: style.css [hunk 3]
```diff
@@ -547,16 +550,72 @@ label {
...
+    display: block;
+    cursor: pointer;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
```
Comment (emmiiorji) [inline style.css]: [Optional] I suggest that you ensure that the mobile version remains responsive until the desktop version.

Design at 613px:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @efmomoh and Team, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Menu appears on click of the hamburger icon
- Menu closes on click of the menu items
- Nice animation for the hamburger


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @efmomoh      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- good commit messages 👍 


## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**